ouroboros-consensus-cardano/src/byron-testlib/Ouroboros/Consensus/ByronDual/Node.hs

```diff
@@ -82,9 +82,11 @@ protocolInfoDualByron :: forall m. Monad m
                       => ByronSpecGenesis
                       -> PBftParams
                       -> [CoreNodeId] -- ^ Are we a core node?
-                      -> ProtocolInfo m DualByronBlock
+                      -> ( ProtocolInfo DualByronBlock
+                         , m [BlockForging m DualByronBlock]
```

Comment:
Now that we return two things I wonder if we should rename this function. 

Also, we should add a comment explaining what the second component of the tuple is, and maybe how it relates to the input parameters.

Reply:
Yeah maybe but that would be a very disruptive change, wouldn't it?